model/field/rum_v3_mapping.go

```diff
@@ -131,13 +133,17 @@ func Mapper(shortFieldNames bool) func(string) string {
 	}
 }
 
+var once sync.Once
+var inverseMap = map[string]string{}
```

Comment:
The `rumV3Mapping` and the `inverseMap` are conceptually the same, the one is just an inversion of the other. Why would it be important that the `inverseMap` is lazy initialized on the first call, instead of also defining it as global variable on loading?

For the sake of consistency and simplicity I suggest to instead do a simple initialization, such as 
```
var inverseMap = func() map[string]string {
	m := make(map[string]string)
	for k, v := range rumV3Mapping {
		m[v] = k
	}
	return m
}()
```

For naming consistency I further suggest to use `rumV3InverseMapping`.

Reply:
sure